packages/boot/src/mixins/boot.mixin.ts

```diff
@@ -104,6 +106,18 @@ export function BootMixin<T extends MixinTarget<Application>>(superClass: T) {
       );
     }
 
+    /**
+     * Register a booter to boot a sub-application
+     * @param subApp - A sub-application with artifacts to be booted
+     * @param filter - A binding filter to select what bindings from the sub
+     * application should be added to the main application.
```

Comment:
Can you please briefly mention what is the default behavior when no filter is provided? Are we going to load all bindings or no bindings?